raiden-dapp/src/locales/en.json

```diff
@@ -468,6 +468,18 @@
         "File a new issue on https://github.com/raiden-network/light-client",
         "Upload logs to the issue"
       ]
+    },
+    "PFS_NO_ROUTES_BETWEEN_NODES": {
+      "title": "No route between you and your target",
+      "problem": "The Pathfinding Service was not able to find a route between you and the target you want to transfer tokens to.",
+      "solution": [
+        "Hit the ← button in the top left",
```

Comment:
Maybe it'd be worth it to, instead of describing each click the user needs to do, just to tell them they may solve it by connecting to a better connected hub? Not sure, just a thought

Reply:
That'd be ok if the average user would know what a hub is. 

Unfortunately we don't do a good job of explaining what it is at the moment and have also identified this in an on-site meeting last year, so I think being explicit here is the right thing to do